src/commands/WeatherCommand.js

```diff
@@ -19,25 +20,27 @@ class WeatherCommand extends Command {
 
     init(bot) {
         this.client = bot.client;
-        this.unit = bot.settings.modules.weather.degreeUnit;
+        this.unit = Config.get("commands.weather.unit");
     }
 
     call(args, message) {
-        Weather.find({search: args.join(" "), degreeType: this.unit}, (err, result) => {
+        Weather.find({ search: args.join(" "), degreeType: this.unit }, (err, result) => {
             if (result[0] != undefined) {
-            
-                const resultBase  = result[0];
+
+                const resultBase = result[0];
                 const resultForToday = resultBase.forecast[1];
                 const resultForTommorow = resultBase.forecast[2];
                 const currentLocation = resultBase.location.name;
                 const degree = " °" + this.unit;
-            
+                const resultTodaySkytext = resultBase.current.skytext.toLowerCase().replace(/ /g, "-");
+                const resultTommorowSkytext = resultBase.current.skytext.toLowerCase().replace(/ /g, "-");
```

Comment:
Why are both skytexts for current (today - current) property? Change line 36 current to tomorrow or whatever the api uses.